railties/test/application/configuration_test.rb

```diff
@@ -1708,7 +1708,28 @@ def index
       app "development"
       ActiveSupport::Dependencies.autoload_paths.each do |path|
         assert_not_operator path, :ends_with?, "app/assets"
-        assert_not_operator path, :ends_with?, "app/#{Rails.configuration.webpacker_path}"
+        assert_not_operator path, :ends_with?, "app/javascript"
+      end
+    end
+
+    test "autoload paths will exclude changed path" do
+      app_file "config/webpacker.yml", <<-YAML
+        default: &default
+          source_path: app/webpack
+          check_yarn_integrity: false
+        development:
+          <<: *default
+        test:
+          <<: *default
+        production:
+          <<: *default
+      YAML
+
+      app "development"
+
+      ActiveSupport::Dependencies.autoload_paths.each do |path|
+        assert_not_operator path, :ends_with?, "app/assets"
+        assert_not_operator path, :ends_with?, "app/webpack"
```

Comment:
How is this test passing? We are not setting the config to webpack anywhere yet. Unless I'm missing something.

Reply:
I added a new commit to this branch that makes this test fail if the config is not set.